### PR TITLE
chapter_4,5

### PR DESCRIPTION
**Pull request by maihuysl1999**
Static Page 
https://docs.google.com/document/d/1L9dlWmhGjGeCMPiZD5of0TF-LqGJJK9PEh6HX9fVXcY/edit
Screenshot from 2021-07-29 15-32-32

### CODE REVIEW

**Comment by maihuysl1999** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "sample app", "#", id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home", "#" %></li>
```
Comment (ngocvt-0484) [inline app/views/layouts/_header.html.erb]: i18n

**Comment by ngocvt-0484** [comment]
folder log cho vô gitignore e nhé

**Comment by maihuysl1999** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**